transports/msmq/routing_content_core_6.partial.md

```diff
@@ -3,6 +3,9 @@ Mapping the logical destination to the physical address containing the queue and
 The preferred way of configuring the physical routing is via the instance mapping file.
 
 
+NOTE: When using instance mapping, the settings will have no effect on **audit/error** queue and **publish/subscribe** (except for subscription messages, subscription messages will be sent to all listed instances)
```

Comment:
shouldn't this be within the instance mapping file section?
should it be spelled out "audit and error queues"?
do we need to mention pubsub?

Reply:
I get why we mention it does not include audit/error but why mentioning pub/sub here?

Reply:
Doesn't it make sense to clarify that when publishing to only the instances that have subscribed will get the event regardless what's present in the mapping file?

Reply:
That way of wording makes more sense than stating that pubsub is t affected except for subscription messages.